Sources/SwiftDocC/Benchmark/Metrics/OutputSize.swift

```diff
@@ -53,20 +53,22 @@ extension MetricValue {
     /// corresponding files. This behavior helps produce real deltas between
     /// multiple benchmarks.
     init?(directory: URL) {
-        guard let enumerator = FileManager.default.enumerator(
-            at: directory,
-            includingPropertiesForKeys: [.totalFileAllocatedSizeKey, .fileAllocatedSizeKey],
-            options: .skipsHiddenFiles,
-            errorHandler: nil
-        ) else {
+        guard
+            let enumerator = FileManager.default.enumerator(
```

Comment:
Are all `if let` and `guard let` statements split up like this?

Reply:
no just for long lines unfortunately.